Makefile

```diff
@@ -1,5 +1,5 @@
 test:
-	go test -v ./...
+	go test -cover -v ./...
```

Comment:
I would have expected this to be in cover rule, so a few line under this one

Reply:
I just got tired of having them be separate in the `Makefil`, so I just included it. It's 99% just for my debug happiness, hah.

Reply:
I would suggest to use tparse locally then

go test -cover -json ./... | tparse

https://github.com/mfridman/tparse

I love it since I discovered

Reply:
@syntaqx I'm unsure if you saw it